### PR TITLE
job_finding_website

### PR DESCRIPTION
**Pull request by Richadxt**
# Description
This is the front page design of the job_finding_website.This website is made with HTML CSS Bootstrap and Javascript.

Fixes # (428.)

## Type of change

# Explain the Testing instructions

**Test Configuration**:

- Operating system: Windows/Linux/Mac
- Version:any
- Text-editors used:Visual Studio Code

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by DEEPALI-SUK — COMMENTED** [review]
Link the issue no properly.

**Comment by Richadxt** [comment]
Actually i,m contributing in the open source first time ever, i don't know how to contribute ...so please tell me what to do next...

File: job_finding_website/test/test.rtf [hunk 1]
```diff
@@ -0,0 +1 @@
+{\rtf1}
```
Comment (arpit456jain) [inline job_finding_website/test/test.rtf:1]: remove this take help from mentors

**Review by arpit456jain — CHANGES_REQUESTED** [review]

File: job_finding_website/test/test.rtf (hunk 1 above)
Comment (Richadxt) [reply job_finding_website/test/test.rtf:1]: I have removed the test folder...

**Pull request closed without merging**